mne/io/pick.py

```diff
@@ -93,6 +93,9 @@ def pick_channels(ch_names, include, exclude=[]):
     sel : array of int
         Indices of good channels.
     """
+    if not all([isinstance(kind, (list, tuple, np.ndarray))
+                for kind in (include, exclude)]):
+        raise AssertionError('include/exclude must be list, tuple, or "bads"')
```

Comment:
Ah yes - I'll add an if statement to check for that. There are a few tests
breaking here and there and I'm trying to fix them now. There were a few
situations where the default was "bads", but it wasn't doing anything, so
now that I changed the behavior to actually exclude bad electrodes it is
breaking.

On Mon, Aug 3, 2015 at 10:32 AM, Alexandre Gramfort <
notifications@github.com> wrote:

> In mne/io/pick.py
> <https://github.com/mne-tools/mne-python/pull/2375#discussion_r36108941>:
>
> > @@ -93,6 +93,9 @@ def pick_channels(ch_names, include, exclude=[]):
> >      sel : array of int
> >          Indices of good channels.
> >      """
> > +    if not all([isinstance(kind, (list, tuple, np.ndarray))
> > +                for kind in (include, exclude)]):
> > +        raise AssertionError('include/exclude must be list, tuple, or "bads"')
>
> exclude cannot be bads here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/2375/files#r36108941>.
>